src/controllers/error.js

```diff
@@ -1 +1,17 @@
+const path = require('path');
```

Comment:
we do not use path in this file

Reply:
this path for 404 error page

Reply:
yeah ,, we required path ,, but it's actually not used yet

Reply:
ok I just set it up to start working in this pages :)